src/test/java/com/microsoft/util/ElementUtilTest.java

```diff
@@ -55,13 +55,23 @@ public void extractPackageElements() {
     public void extractSortedElements() {
         Element element = elements.getPackageElement("com.microsoft.samples.subpackage");
 
-        List<String> result = elementUtil.extractSortedElements(element)
+        List<String> allElements = element.getEnclosedElements()
+                .stream().map(String::valueOf).collect(Collectors.toList());

```

Comment:
.stream().map(String::valueOf).collect(Collectors.toList()); 
what's each operation for? looks complex here.

Reply:
The purpose is get enclosed element names as List<String>.

`.stream().map(String::valueOf)` -> applies toString() on each element of stream
`collect(Collectors.toList())` -> transformed stream to List<String>